benchmarks/std/containers/sequences/vector.bool/copy_n/test.cpp

```diff
@@ -0,0 +1,101 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#include <benchmark/benchmark.h>
+//
+#include <algorithm>
+#include <functional>
+#include <random>
+#include <vector>
+
+using namespace std;
+
+#pragma warning(disable : 4365)
+
+inline static vector<bool> createRandomVector(const size_t size) {
+    static random_device rd;
+    static mt19937 gen{random_device{}()};
+    vector<bool> result(size);
+    generate_n(result.begin(), size, []() { return bernoulli_distribution{0.5}(gen); });

```

Comment:
unnecessary `()`

Reply:
I am not sure, I thought that this is only C++20 and I would want to be able to run the tests in earlier standard modes too

Reply:
It has been there since lambdas introduction: https://godbolt.org/z/7cnWEMa6G

Reply:
If we remove `()` here we should on line 17 in the previous file as well.